index.bs

```diff
@@ -4222,6 +4234,43 @@ To <dfn export>run WebDriver BiDi preload scripts</dfn> given |environment setti
 
 ### Types ### {#module-script-types}
 
+#### The script.Channel type #### {#type-script-Channel}
+
+<pre class="cddl remote-cddl local-cddl">
+script.Channel = {
+  type: "channel",
+  value: script.ChannelProperties,
+};
+
+script.ChannelId = text;
+
+script.ChannelProperties = {
+  channel: script.ChannelId,
+  ?maxDepth: js-uint,
```

Comment:
We don't support `maxDepth > 1` yet, and the work on it is not planned yet, so let's either remove it and default to `1` like `log.entryAdded` and `script.callFunction`.

Reply:
It does default to 1 (not in the CDDL but in the prose).

Reply:
Sorry, didn't finish the sentence. Let's either remove it, or restrict the value to be in `[0,1]`

Reply:
Addressed https://github.com/w3c/webdriver-bidi/issues/390

Reply:
Addressed https://github.com/w3c/webdriver-bidi/issues/390